christfellowshipchurch/web-app-v2
# Blob breaking Add to Calendar - removing temporarily

### About
The blob method that allows for icsLink to work on Safari is breaking Add to Calendar on the rest of the browsers, so I'm removing it temporarily 

### Test Instructions
Open any event and Add to Calendar - Apple or Outlook should be working on any browser but Safari

### Closes Tickets


### Related Tickets
N/A
